src/app/screens/BookDetail/index.tsx

```diff
@@ -10,11 +12,46 @@ interface Props {
   };
 }
 
+interface dataComments {
+  item: {
+    id: number;
+    name: string;
+    lastname: string;
+    comment: string;
+    avatarImage: string | null;
```

Comment:
you set this props to any in Comment. Change Comment prop type

Reply:
The error was in the "any" declaration, i changed that.